runtests.py

```diff
@@ -554,7 +558,7 @@ def build_suite(self):
                     continue
                 suite.addTest(
                     self.handle_directory(path, filename))
-        if sys.platform not in ['win32']:
+        if sys.platform not in ['win32'] and self.add_embedded_test:
```

Comment:
Hmm, this embed-test integration really is a hack. And excluding it with this flag makes it even worse. If anything, then _including_ the test should be done explicitly, not _excluding_ it.

Reply:
Done